frontend/src/app/Settings/types.d.ts

```diff
@@ -183,3 +183,18 @@ interface FacilityData {
 }
 
 type TestCorrectionStatus = "ORIGINAL" | "CORRECTED" | "REMOVED";
+
+interface FacilitiesState {
+  current: Facility | null;
+  list: Facility[];
```

Comment:
```suggestion
  selectedFacility: Facility | null;
  availableFacilities: Facility[];
```

`list` and `current` are ambiguous names that are hard to parse within the components.

Reply:
good point, thank you. will change.